src/TraceEvent/DynamicTraceEventParser.cs

```diff
@@ -1659,13 +1876,13 @@ public void ToStream(Serializer serializer)
             {
                 serializer.Write((short)Offset);
                 serializer.Write((short)Size);
-                if (Type == null)
+                if (Type is not { } t)
```

Comment:
Are these changes required for NativeAOT, or is this just using newer syntax?

Reply:
That particular line is just using newer syntax to avoid any potential race conditions between field reads.